lib/request.js

```diff
@@ -58,11 +58,14 @@ var request = function (params, body) {
       return err;
     })
     .then(function (response) {
+      var contentType = response.headers['content-type'] || [];
+      var requestId = response.headers['X-Request-ID'] || null;
```

Comment:
The reason phantomjs tests are failing is that in this particular environment, `response.headers` is undefined so you're basically getting a KeyError. The correct snippet should look somewhat like:

```
      var contentType = (response.headers || {})['content-type'] || [];
      var requestId = (response.headers || {})['X-Request-ID'] || null;
```

The other question however is why phantomjs didn't output the error to console and I'm going to fix that in the next PR.